redaxo/src/core/pages/system.log.external.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @package redaxo5
+ */
+
+$func = rex_request('func', 'string');
+$logFile = ini_get('error_log');
+/*
+$logFile = rex_request('file', 'string');
+$availableLogs = [ini_get('error_log')];
+
+if (!in_array($logFile, $availableLogs)) {
+    unset($logFile);
+}
+*/
+
+if (!isset($logFile) || filesize($logFile) <= 0) {
+    return;
+}
+
+$content = '
+            <table class="table table-hover">
+                <tbody>';
+
+$file = new SplFileObject($logFile, 'r');
+$file->seek(PHP_INT_MAX);
+$last_line = $file->key();
+
+$limit = 30;
+$lines = iterator_to_array(new LimitIterator($file, max(0, $last_line - $limit), $last_line));
```

Comment:
Alternativ könnte man auch überlegen, irgendwie die Logik aus `rex_log_file` wiederzuverwenden:
https://github.com/redaxo/redaxo/blob/master/redaxo/src/core/lib/util/log_file.php#L76-L79

Die Variante hat den Vorteil, dass Leerzeilen zum Beispiel direkt ausgelassen werden. Wobei ich nicht weiß, ob in den Systemlogdateien überhaupt Leerzeilen vorkommen, daher ist dieser Gesichtspunkt vielleicht auch unwichtig.
Aktuell wird hier aber ja erst limitiert, und dann noch Leerzeilen rausgelöscht.

Ich vermute mal, dass die aktuelle Variante alle Zeilen durchgeht, von Anfang an. Und der LimitIterator beginnt dann aber halt erst mit der Ausgabe nachdem alles iteriert wurde und klar ist, welche Element ausgegeben werden müssen. Wobei Performance hier jetzt auch nicht so wirklich wichtig ist!

Reply:
würde ich als separaten PR sehen.